src/backend/api/Fusion.Resources.Api/Controllers/Requests/DepartmentRequestsController.cs

```diff
@@ -39,5 +42,59 @@ public class DepartmentRequestsController : ResourceControllerBase
             var apiModel = result.Select(x => new ApiResourceAllocationRequest(x)).ToList();
             return new ApiCollection<ApiResourceAllocationRequest>(apiModel);
         }
+
+        [HttpGet("departments/{departmentString}/resources/requests/timeline")]
+        public async Task<ActionResult<ApiRequestsTimeline>> GetDepartmentTimeline(
+            [FromRoute] string departmentString,
+            [FromQuery] ODataQueryParams query,
+            [FromQuery] DateTime? timelineStart = null,
+            [FromQuery] string? timelineDuration = null,
+            [FromQuery] DateTime? timelineEnd = null)
+        {
+            #region Authorization
+
+            var authResult = await Request.RequireAuthorizationAsync(r =>
+            {
+                r.AlwaysAccessWhen().FullControl().FullControlInternal();
+                r.AnyOf(or =>
+                {
+                    // add requirements
```

Comment:
Is this still to be decided?

Reply:
The authorization? Yes will implement that later. For now we just give admin access.